src/coff_import_file.rs

```diff
@@ -3,7 +3,898 @@
+            pointer_to_raw_data: u32!((size_of::<ImageFileHeader>()
+                + NUMBER_OF_SECTIONS * size_of::<ImageSectionHeader>())
+            .try_into()
+            .unwrap()),
+            pointer_to_relocations: u32!(0),
+            pointer_to_linenumbers: u32!(0),
+            number_of_relocations: u16!(0),
+            number_of_linenumbers: u16!(0),
+            characteristics: u32!(
+                IMAGE_SCN_ALIGN_4BYTES
+                    | IMAGE_SCN_CNT_INITIALIZED_DATA
+                    | IMAGE_SCN_MEM_READ
+                    | IMAGE_SCN_MEM_WRITE
+            ),
+        }];
+        buffer.write_all(unsafe { any_as_u8_slice(&section_table) })?;
+
+        // .idata$3
+        let import_descriptor = ImageImportDescriptor {
+            original_first_thunk: u32!(0),
+            time_date_stamp: u32!(0),
+            forwarder_chain: u32!(0),
+            name: u32!(0),
+            first_thunk: u32!(0),
+        };
+        buffer.write_all(unsafe { any_as_u8_slice(&import_descriptor) })?;
+
+        // Symbol Table
+        let mut symbol_table: [_; NUMBER_OF_SYMBOLS] = [ImageSymbol {
+            name: [0; 8],
+            value: u32!(0),
+            section_number: u16!(1),
+            typ: u16!(0),
+            storage_class: IMAGE_SYM_CLASS_EXTERNAL,
+            number_of_aux_symbols: 0,
+        }];
+        set_name_to_string_table_entry(&mut symbol_table[0], size_of::<u32>());
+        buffer.write_all(unsafe { any_as_u8_slice(&symbol_table) })?;
+
+        // String Table
+        write_string_table(&mut buffer, &[NULL_IMPORT_DESCRIPTOR_SYMBOL_NAME])?;
+
+        Ok(NewArchiveMember::new(
+            buffer.into_boxed_slice(),
+            &DEFAULT_OBJECT_READER,
+            self.import_name.to_string(),
+        ))
+    }
+
+    /// Create a NULL Thunk Entry.  This is a small object file which contains a
+    /// NULL Import Address Table entry and a NULL Import Lookup Table Entry.  It
+    /// is used to terminate the IAT and ILT.
+    fn create_null_thunk(&self) -> Result<NewArchiveMember> {
+        let mut buffer = Vec::new();
+
+        const NUMBER_OF_SECTIONS: usize = 2;
+        const NUMBER_OF_SYMBOLS: usize = 1;
+        let va_size = if self.is_64_bit() { 8 } else { 4 };
+
+        // COFF Header
+        let header = ImageFileHeader {
+      machine: u16!(self.native_machine.into()),
+      number_of_sections: u16!(NUMBER_OF_SECTIONS.try_into().unwrap()),
+      time_date_stamp: u32!(0),
+      pointer_to_symbol_table: u32!((size_of::<ImageFileHeader>() + (NUMBER_OF_SECTIONS * size_of::<ImageSectionHeader>()) +
+          // .idata$5
+          va_size +
+          // .idata$4
+          va_size).try_into().unwrap()),
+          number_of_symbols: u32!(NUMBER_OF_SYMBOLS.try_into().unwrap()),
+          size_of_optional_header: u16!(0),
+          characteristics: u16!(if self.is_64_bit() { 0 } else { IMAGE_FILE_32BIT_MACHINE }),
+  };
```

Comment:
Looks like rustfmt gave up here due to the pointer_to_symbol_table field. Can you comment it out, apply rustfmt and then uncomment it again and try to manually format that field?